variables.tf

```
@@ -328,3 +328,9 @@ variable "serverless_cache_usage_limits" {
   default     = {}
   description = "The usage limits for the serverless cache"
 }
+
+variable "serverless_snapshot_arns_to_restore" {
+  type        = list(string)
+  default     = []
+  description = "Optional list of snapshot ARNs to restore for serverless ElastiCache."
```

This might be a nitpick but could we update this description to something like:

```suggestion
  description = "The list of ARN(s) of the snapshot that the new serverless cache will be created from. Available for Redis only."
```